widget/embedded/src/store/slices/data.ts

```diff
@@ -155,12 +155,22 @@ export const createDataSlice: StateCreator<
           }
         }
 
-        //3. sort by token popularity
+        const blockChainA = blockchainsMapByName.get(tokenA.blockchain);
+        const blockChainB = blockchainsMapByName.get(tokenB.blockchain);
```

Comment:
I'm not familiar enough with sort tokens. It's better to check with @nikaaru

Reply:
done